servicenow.json

```diff
@@ -173,764 +178,731 @@
                 },
                 {
-                    "data_path": "action_result.data.*.invoice_number",
-                    "data_type": "string"
-                },
-                {
-                    "data_path": "action_result.data.*.model_category.link",
+                    "data_path": "action_result.data.*.incident_state",
                     "data_type": "string",
                     "example_values": [
-                        "https://dev78070.service-now.com/api/now/table/cmdb_model_category/35bf2d4137101000deeabfc8bcbe5dbd"
+                        "Closed"
                     ]
                 },
                 {
-                    "data_path": "action_result.data.*.model_category.display_value",
+                    "data_path": "action_result.data.*.install_date",
                     "data_type": "string",
                     "example_values": [
-                        "Software License"
+                        "2019-08-19 01:00:00"
                     ]
                 },
                 {
-                    "data_path": "action_result.data.*.disposal_reason",
-                    "data_type": "string"
+                    "data_path": "action_result.data.*.install_status",
+                    "data_type": "string",
+                    "example_values": [
+                        "In use"
+                    ]
                 },
                 {
-                    "data_path": "action_result.data.*.retirement_date",
+                    "data_path": "action_result.data.*.invoice_number",
                     "data_type": "string"
                 },
                 {
-                    "data_path": "action_result.data.*.expenditure_type",
-                    "data_type": "string"
+                    "data_path": "action_result.data.*.is_merged_license",
+                    "data_type": "string",
+                    "example_values": [
+                        "false"
+                    ]
                 },
                 {
-                    "data_path": "action_result.data.*.depreciation_date",
+                    "data_path": "action_result.data.*.justification",
                     "data_type": "string"
                 },
                 {
-                    "data_path": "action_result.data.*.is_merged_license",
+                    "data_path": "action_result.data.*.knowledge",
                     "data_type": "string",
                     "example_values": [
                         "false"
                     ]
                 },
                 {
-                    "data_path": "action_result.data.*.acquisition_method",
+                    "data_path": "action_result.data.*.lease_id",
                     "data_type": "string"
                 },
                 {
-                    "data_path": "action_result.data.*.assigned_condition",
+                    "data_path": "action_result.data.*.license_key",
                     "data_type": "string"
```

Comment:
Should any of these fields have "contains"?

Reply:
I am not sure about these fields currently. I have worked only on the on_poll action part.